src/test/kotlin/io/bazel/kotlin/builder/tasks/jvm/KotlinKSPMetaFilesCopyTest.kt

```diff
@@ -0,0 +1,30 @@
+package io.bazel.kotlin.builder.tasks.jvm
+
+import io.bazel.kotlin.KotlinAssertionTestCase
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+@RunWith(JUnit4::class)
+class KotlinKSPMetaFilesCopyTest: KotlinAssertionTestCase("src/test/data/jvm/ksp") {
+
+    @Test
+    fun testKSPCopiesAllFilesFromMetaINF() {
+        jarTestCase("coffee_lib.jar", description = "Generated jar with ksp plugin contains meta-inf contents") {
+            assertContainsEntries("META-INF/", "META-INF/MAIFEST.MF")
+        }
+    }
+
+    @Test
+    fun testKSPCopiesAllFilesFromMetaINFInMoshiLib() {
+        jarTestCase("moshi_lib.jar", description = "Generated jar with ksp plugin contains meta-inf contents") {
+            assertContainsEntries(
```

Comment:
Will this test start failing if there are things under the META-INF directory that aren't expected?

Reply:
No, if there are additional files, the test will still pass. I did verify that.

Reply:
Might be worth testing that case as well to just be sure that unexpected stuff isn't leaking into the directory.

Reply:
changed to verify exact match of file names